.github/workflows/publish.yml

```diff
@@ -16,15 +16,17 @@ jobs:
     - uses: actions/setup-dotnet@v3
       with:
         dotnet-version: 6.0.100
-    - name: bulid
+    - name: build
       run: |
         dotnet_args="-c Release -p:NoPackageAnalysis=true"
         if [[ ! "$GITHUB_REF" =~ ^refs/tags/* ]]; then
           project_suffix=dev.${{ github.sha }}
           dotnet_args="$dotnet_args --version-suffix $project_suffix"
         fi
-        dotnet build "$dotnet_args"
-        dotnet pack "$dotnet_args"
+        # shellcheck disable=SC2086
+        dotnet build $dotnet_args
+        # shellcheck disable=SC2086
+        dotnet pack $dotnet_args
```

Comment:
In the shellcheck wiki, it suggests using like `${dotnet_args}` 🤔 But the current version can be merged too! 👍🏻 

https://github.com/planetarium/lib9c/issues/2103